src/commands/dockersupport/localDockerSupport.ts

```diff
@@ -14,51 +17,39 @@ import { validateDockerInstalled } from "./validateDockerInstalled";
  * @param node - Function App Project
  */
 export async function localDockerPrompt(context: IActionContext, devContainerFolderPathUri: vscode.Uri, node?: SlotTreeItemBase, devContainerName?: string): Promise<void> {
-    if (node) {
-        // check if language is supported for dev container
-        if (devContainerName) {
-            // check if the function app is in Linux
-            if (node.site.reserved) {
-                // asks if the user wants to use Docker for initializing the project locally
-                const useDocker: string = await prompt(context);
-                if (useDocker === "yes") {
-                    // download dev container
-                    await requestUtils.downloadFile(
-                        `https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/containers/${devContainerName}/.devcontainer/devcontainer.json`,
-                        vscode.Uri.joinPath(devContainerFolderPathUri, 'devcontainer.json').fsPath
-                    );
-                    // download docker file
-                    await requestUtils.downloadFile(
-                        `https://raw.githubusercontent.com/microsoft/vscode-dev-containers/master/containers/${devContainerName}/.devcontainer/Dockerfile`,
-                        vscode.Uri.joinPath(devContainerFolderPathUri, 'Dockerfile').fsPath
-                    );
-                    // external - check if Docker is installed, Remote Development extension AND Docker Extension
-                    if (!validateDockerInstalled()) {
-                        // if Docker is not downloaded - ask the user if they'd like to download Docker
-                        const downloadDocker: string = await prompt2(context);
-                        if (downloadDocker === "yes") {
-                            // Check if Operating System is Windows
-                            if (process.platform == "win32") {
-                                // Download Docker with an MSI package
-                            } else {
-                                // Not windows: display link to download docker externally from Docker documentation
-                                void vscode.window.showInformationMessage(localize('downloadDocker', 'Check the (Docker documentation)[https://docs.docker.com/get-docker/] to download Docker for your system'));
-                            }
-                        } else {
-                            void vscode.window.showInformationMessage(localize('noDocker', 'Continuing without the use of Docker as user requested'));
-                        }
+    if (node && devContainerName && node.site.reserved) {
+        // asks if the user wants to use Docker for initializing the project locally
+        if (await prompt(context) === DOCKER_PROMPT_YES) {
```

Comment:
actually on the Fadi's PR I noticed there was `DialogResponse.yes` is that available to use here? if so, lets use that instead of creating a new one.

Reply:
It's not possible since `DialogResponse.yes` is of type `MessageItem` and it cannot overlap with the `string` type

Reply:
Is this now fine to leave as is?

Reply:
yup.. its fine to leave as is.